### PR TITLE
Adds a method to ReactMeteorDataComponent for retrieving the wrapped …

### PR DESCRIPTION
**Pull request by rooch84**
…component.

This is useful if the parent component needs to call a method on the child.

### CODE REVIEW

File: packages/react-meteor-data/ReactMeteorData.jsx [hunk 1]
```diff
@@ -180,6 +180,9 @@ export default function connect(options) {
       getMeteorData() {
         return getMeteorData(this.props);
       }
+			getWrappedComponent() {
```
Comment (hwillson) [inline packages/react-meteor-data/ReactMeteorData.jsx]: Would you mind fixing the spacing here to align with the style used in the rest of the file?

**Comment by yched** [comment]
This would be fixed by #266 I guess.

**Comment by filipenevola** [comment]
On version 2.0.0 a lot of things were changed then if you think this PR is still valid please update it and open again. Thanks.

**Comment by derwaldgeist** [comment]
Has this been implemented?

**Comment by yched** [comment]
@derwaldgeist in the current release, withTracker() uses React.forwardRef, so you should be able to get the wrapped component symply by using a regular React ref

**Comment by derwaldgeist** [comment]
@yched Thanks for your feedback. Yeah, I realized this in the meantime. I was still using version 1.0.0 of this package so far, where it did not work. Had to update Meteor to 1.10 though, because of version dependencies on the ecmascript package.

**Pull request closed without merging**